lib/ash_authentication/user_identity/upsert_identity_change.ex

```diff
@@ -38,6 +38,7 @@ defmodule AshAuthentication.UserIdentity.UpsertIdentityChange do
       |> Map.values()
       |> Enum.reject(&is_nil/1)
       |> List.first()
+      |> convert()
```

Comment:
I think we can just to `to_string()` here since that doesn’t do anything if it is already a string.